test/browser/package.json

```diff
@@ -10,7 +10,9 @@
     "test:safaridriver": "PROVIDER=webdriverio BROWSER=safari pnpm run test:unit",
     "test-fixtures": "vitest",
     "test-mocking": "vitest --root ./fixtures/mocking",
-    "coverage": "vitest --coverage.enabled --coverage.provider=istanbul --browser.headless=yes"
+    "coverage": "vitest --coverage.enabled --coverage.provider=istanbul --browser.headless=yes",
+    "test:browser:playwright": "PROVIDER=playwright vitest --ui",
```

Comment:
1. there is already test-fixtures, adding PROVIDER= is not that longer
2. why --ui? it's already built in

Reply:
I haven't playwright installed, I use webdriverio :sweat_smile: :

![imagen](https://github.com/vitest-dev/vitest/assets/6311119/6bb7e364-6129-482d-a891-c6d662fac472)

Reply:
This way we can run the script

Reply:
removing the `--ui` flag

Reply:
ah, you are saying that on windows it's harder to set env

Reply:
yes, I neet to use `set PROVIDER=xxxx`